devices/oneplus-oneplus2/default.nix

```diff
@@ -0,0 +1,58 @@
+{ config, lib, pkgs, ... }:
+
+{
+  mobile.device.name = "oneplus-oneplus2";
+  mobile.device.identity = {
+    name = "OnePlus 2";
+    manufacturer = "OnePlus";
+  };
+
+  mobile.hardware = {
+    soc = "qualcomm-msm8994";
+    # FIXME(Krey): This devices also has 3GB LPDDR4 version
+    ram = 1024 * 4;
```

Comment:
This is purely informational. The smallest capacity officially sold should be used. Though to be fair it's not a big deal as it's still not being used.

Reply:
Should i integrate something like an option to specify the capacity then? e.g. I have hardware modifications on mine that may allow for non-standard capacity

Reply:
Nope, this is purely informational, and **if** it ever ends up being used, it will be to select different defaults based on the hardware.

If it ever becomes an issue, as the end-user, you could add `mobile.hardware.ram = lib.mkForce 8 * 1024;` (arbitrary number, I don't know your specific situation).

Though it'll probably never become an issue. I don't have any uses planned for this feature.